apps/o2-blocks/src/github-issue-template/editor.js

```diff
@@ -84,15 +79,14 @@ const Invalid = () => (
 		className="is-warning"
 		title={ __( 'Please fill in the required fields' ) }
 		subTitle={ __(
-			'Title, Org, and Repository are required. Select this block to open the form and fill them.'
+			'Org, and Repository are required. Select this block to open the form and fill them.'
 		) }
-		icon="⚠"
 	/>
 );
 
 registerBlockType( 'a8c/github-issue-template', {
 	title: __( 'Github Issue Template', 'a8c' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **7** times:
`translate( 'Github Issue Template' )` *ES Score: 12*